### PR TITLE
PLAT-143787: Added screenshot tests for TabLayout

### PR DESCRIPTION
**Pull request by paul-beldean-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added extra screenshot tests for TabLayout

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-143787

### Comments

Enact-DCO-1.0-Signed-off-by: Paul Beldean paul.beldean@lgepartner.com

### CODE REVIEW

**Comment by adrian-cocoara-lgp** [comment]
Looks good, but some tests for RTL locale would be nice 😅

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

File: tests/screenshot/apps/components/TabLayout.js [hunk 1]
```diff
@@ -90,13 +105,97 @@ const TabLayoutTests = [
...
+	{
+		component: <TabLayout anchorTo="end">{tabs}</TabLayout>,
+		wrapper: {full: true}
+	},
+	{
+		component: <TabLayout
+			dimensions={{
+				tabs: {
+					collapsed: 300,
+					normal: 900
```
Comment (dongsuwon) [inline tests/screenshot/apps/components/TabLayout.js:142]: Result of this test title is overlapped with focused tap. Is this the intended test?

File: tests/screenshot/apps/components/TabLayout.js [hunk 2]
```diff
@@ -90,13 +105,97 @@ const TabLayoutTests = [
...
+			}}
+		>{tabs}</TabLayout>,
+		wrapper: {full: true}
+	},
+	{
+		component: <TabLayout
+			dimensions={{
+				tabs: {
+					collapsed: 300,
+					normal: 800
```
Comment (dongsuwon) [inline tests/screenshot/apps/components/TabLayout.js:157]: same above

File: tests/screenshot/apps/components/TabLayout.js (hunk 1 above)
Comment (paul-beldean-lgp) [reply tests/screenshot/apps/components/TabLayout.js:142]: It was more intended to check for different normal values to catch if it displays differently. (the same for the bellow one)

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Comment by dongsuwon** [comment]
LGTM

**Pull request merged**